http/http.go

```diff
@@ -66,6 +66,8 @@ func (s *Service) get(ctx context.Context, endpoint string) (io.Reader, error) {
 		return nil, errors.Wrap(err, "failed to create GET request")
 	}
 	req.Header.Set("Accept", "application/json")
+	s.addExtraHeaders(req)
```

Comment:
Please do this prior to the `req.Header.Set()` operations that are required, to avoid bad extra headers overwriting those set by the module (here and wherever else you do this).

Reply:
Fixed.